README.md

```diff
@@ -7,3 +7,19 @@ A live version of the app is available here: https://vberger.github.io/Bayes-O-M
 An explanation of what Bayesian Networks are, how they can be used, and how to use
 the app is available in-app via the "Help" button, or as a
 [markdown file](https://github.com/vberger/Bayes-O-Matic/tree/master/static/help).
+
+
+
+## How to install
+
+### Prerequisites
+
+Before start installation, be sure the *C++ tools* is installed on your system. You can download it into the [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16).
```

Comment:
```suggestion
You will need to install the compiler for the Rust programming language. Do to so there are some prerequisites:

- On Windows, you need the *C++ tools* is installed on your system. You can download it into the [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16).
- On Linux you'll need a working compilation toolchain. Your distribution likely provides the necessary package (for example `build-essential` on Debian).
- 
```

Reply:
I'd rather formulate it like so, to be more cross-platform. I don't know if there are prerequisites for macos though.